dashboard/static/src/js/components/Table.js

```diff
@@ -44,10 +46,10 @@ export default class Table extends React.Component {
             });
         });
       }
-      this.setState({ data: dataSet, total: dataSet.length, loading: false });
-    }, function(error) {
-      console.error("Failed!", error);
-      this.setState({loading: false});
+      this.setState({ data: dataSet, total: dataSet.length, loading: false, hasError: false });
+    }, error => {
```

Comment:
@achen2401 can we switch on the status_code?  it would be nice if a 401 could display something like "Inadequate permissions; have you obtained the necessary roles needed for dashboard access?"

Reply:
@pbugni paul, I am currently getting a **500** server error, not a 401 when I tried accessing the dashboard with insufficient role permission.  I added a more informative message though to let user that permission might be an issue.